backend/src/db/migrations/20230323090957_add_sections_table.ts

```diff
@@ -0,0 +1,33 @@
+import { type Knex } from 'knex';
+
+const TABLE_NAME = 'sections';
+
+const ColumnName = {
+  ID: 'id',
+  NAME: 'name',
```

Comment:
```suggestion
  NAME: 'name',
  TYPE: 'type',
```
it should be an enum data type. for now you can add only two values - `header` and `footer`. we will add more soon
and add it to all stuff (types, model, etc)

Reply:
no, we need to add new column - `type` (don't forget to add it to the db schema). it should be a enum